pkg/provider/gitea/gitea.go

```diff
@@ -110,6 +110,13 @@ func (v *Provider) CreateStatus(_ context.Context, _ versioned.Interface, event
 	case "skipped":
 		statusOpts.Title = "Skipped"
 		statusOpts.Summary = "is skipping this commit."
+	case "pending":
+		// for concurrency set title as pending
+		if statusOpts.Title == "" {
+			statusOpts.Title = "Pending"
```

Comment:
this is in case of concurrency? then how abt "Queued"?
or is it pending for triggering?

Reply:
Yes this is for concurrency 
It will display like below

![Screenshot from 2023-06-09 11-46-11](https://github.com/openshift-pipelines/pipelines-as-code/assets/9441662/e5795a24-809d-47f6-a035-417727ff2b20)

Reply:
yeah .do we want to update the message?
`statusOpts.Summary = "is skipping this commit."`
may be `waiting for approval` now ?

Reply:
waiting for approval sounds a good idea

and @savitaashture please get in touch with the rhtap folks since they are using the skipped status on there ans if that would break their flow, ie:

![image](https://github.com/openshift-pipelines/pipelines-as-code/assets/98980/8e3bb5d9-7801-4286-a2c9-d320b276272e)

Reply:
Sure will inform about this with RHTAP foks

Reply:
@chmouel @sm43 this is how it looks after adding that info

![Screenshot from 2023-06-12 19-48-47](https://github.com/openshift-pipelines/pipelines-as-code/assets/9441662/b9e97122-7719-4ab3-b8c5-569d764abf7d)